src/ui/src/abstractlogview.cpp

```diff
@@ -91,7 +91,6 @@
 #include "highlighterset.h"
 #include "highlightersmenu.h"
 #include "log.h"
-#include "logmainview.h"
```

Comment:
abstractlogview.cpp:94: warning: Included header logmainview.h is not used directly (fix available)